frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -306,3 +307,25 @@ export const knativeCamelKameletBindingResourceWatchers = (
     },
   };
 };
+
+export const getTrafficByRevision = (revName: string, service: K8sResourceKind) => {
+  if (!(service.status?.traffic?.length > 0)) {
```

Comment:
Just some more readable suggestions. Upto you if you want to change.
```suggestion
  if (!service.status?.traffic?.length)) {
```
or
```suggestion
  if (_.isEmpty(service.status?.traffic)) {
```

Reply:
Thanks @divyanshiGupta, make sense. Have updated PTAL.